core/server/master/src/test/java/alluxio/master/file/DefaultFileSystemMasterTest.java

```diff
@@ -1570,9 +1571,9 @@ private void startServices() throws Exception {
     mRegistry = new MasterRegistry();
     JournalFactory factory = new MutableJournal.Factory(new URI(mJournalFolder));
     mBlockMaster = new BlockMaster(mRegistry, factory);
-    mExecutorService =
-        Executors.newFixedThreadPool(2, ThreadFactoryUtils.build("FileSystemMasterTest-%d", true));
-    mFileSystemMaster = new FileSystemMaster(mRegistry, factory,
+    mExecutorService = Executors
+        .newFixedThreadPool(2, ThreadFactoryUtils.build("DefaultFileSystemMasterTest-%d", true));
+    mFileSystemMaster = new DefaultFileSystemMaster(mRegistry, factory,
```

Comment:
I looked at the test and I see little reason (two assertions in `stop()`) to set the executor service explicitly ... I suggest we remove the assertions and use the existing `FileSystemMasterFactory.create()` method